app/src/main/java/com/example/mobilelibrary/models/TranslatedText.java

```diff
@@ -0,0 +1,41 @@
+package com.example.mobilelibrary.models;
+
+public class TranslatedText {
+    private String userId;
+    private String textToTranslate;
+    private String translatedText;
+
```

Comment:
Would be nice to add the languages as well traslatedFromLanguage and translatedToLanguage